modules/@apostrophecms/i18n/index.js

```diff
@@ -757,7 +763,7 @@ module.exports = {
         }
         req.baseUrlWithPrefix = `${req.baseUrl}${self.apos.prefix}`;
         req.absoluteUrl = req.baseUrlWithPrefix + req.url;
-        req.prefix = `${req.baseUrlWithPrefix}${self.locales[req.locale].prefix || ''}`;
+        req.prefix = `${req.baseUrlWithPrefix}${self.locales[req.locale]?.prefix || ''}`;
```

Comment:
Something is wrong if the request managed to reach this point with an incorrect `req.locale`. I think the i18n middleware already throws an invalid error. Why if this needed?

Reply:
It breaks tasks for no reason.